src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -345,7 +345,23 @@ bool LottieLoader::frame(float no)
 
 float LottieLoader::totalFrame()
 {
-    return frameCnt;
+    if(this->selectedSection==NONE)
+        return frameCnt;
+    if(!this->comp)
+        sync();
+    if(this->selectedSection==MARKER)
```

Comment:
`this->` is not mandatory in this context. Would be better having `this` as less as possible.